RecoTracker/LST/plugins/alpaka/LSTProducer.cc

```diff
@@ -59,11 +85,12 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
     edm::EDGetTokenT<LSTPhase2OTHitsInput> lstPhase2OTHitsInputToken_;
     edm::EDPutTokenT<LSTOutput> lstOutputToken_;
 
-    //SDL sdl_;
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
Maybe remove these `ifdef`, given that (for now) we specify `<flags ALPAKA_BACKENDS="cuda"/>` in the BuildFile?